src/app/containers/Article/index.jsx

```diff
@@ -11,17 +11,53 @@ import ArticleMain from '../ArticleMain';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import ConsentBanner from '../ConsentBanner';
+import { GhostWrapper, GridItemConstrainedMedium } from '../../lib/styledGrid';
 
 const logger = nodeLogger(__filename);
 
-/*
-  [1] This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
-*/
+const Container = ({ children, service }) => (
+  <Fragment>
+    <GlobalStyle />
+    <ServiceContextProvider service={service}>
+      <Helmet>
+        <link rel="manifest" href={`/${service}/articles/manifest.json`} />
+      </Helmet>
+      <ConsentBanner />
+      <HeaderContainer />
+      {children}
+    </ServiceContextProvider>
+  </Fragment>
+);
+
+Container.propTypes = {
+  children: node.isRequired,
+  service: string.isRequired,
+};
+
 const ArticleContainer = ({ loading, error, data, bbcOrigin }) => {
-  if (loading) return 'Loading...'; /* [1] */
+  /*
+    This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
+  */
+
+  if (loading)
+    return (
+      <Container service="news">
+        <main role="main">
+          <GhostWrapper>
+            <GridItemConstrainedMedium />
+          </GhostWrapper>
+        </main>
+      </Container>
+    );
+
   if (error) {
     logger.error(error);
-    return 'Something went wrong :(';
+    return (
+      <Container service="news">
+        <ErrorMain status={500} />
+        <FooterContainer />
+      </Container>
+    );
   }
 
   if (data) {
```

Comment:
Curious why you havent used `Container` for the data return

Reply:
The loading and error containers shouldn't use use RequestContextProvider. I can move the providers around to ensure we can re-use the same container... let me do that.